include/fc/io/datastream.hpp

```diff
@@ -37,7 +37,7 @@ class datastream {
       }
 
       inline bool write( const char* d, size_t s ) {
-        if( _end - _pos >= (int32_t)s ) {
+        if( size_t(_end - _pos) >= (size_t)s ) {
```

Comment:
Why type-cast `s` to `size_t` when that is already declared in the signature? Just curious...

Reply:
Only to make it match `read`.  Normally would not have done that, but didn't want to change `read`